scripts/utils/ios-tools.ts

```diff
@@ -0,0 +1,19 @@
+import { fileExists, quietExec } from './common'
+
+/**
+ * Finds llvm-objcopy in Homebrew.
+ */
+export async function getObjcopyPath(): Promise<string> {
+  const whichPath = await quietExec('which', ['llvm-objcopy']).catch(() => {})
+  if (whichPath != null) return whichPath
+
+  const paths = [
+    '/opt/homebrew/opt/llvm/bin/llvm-objcopy',
+    '/usr/local/opt/llvm/bin/llvm-objcopy' // for Intel Macs
+  ]
+  for (const path of paths) {
+    if (await fileExists(path)) return path
+  }
+
+  throw new Error('Please install `llvm-objcopy` using `brew install llvm`.')
```

Comment:
Why wouljdn't `which llvm-objcopy` work? A comment to explain why here would be useful. 

Also we should include `brew install llvm` in the README's build instructions